src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -21,8 +20,29 @@ public LineController(final LineService lineService) {
     }
 
     @PostMapping
-    public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
기본 값이 `json`이므로 `MediaType`은 지정 안하셔도 됩니다 😄 
참고만 해주시면 감사하겠습니다 🙇 
```suggestion
    @GetMapping
```